runtime/doc/treesitter.txt

```diff
@@ -945,6 +945,23 @@ parse({lang}, {query})                          *vim.treesitter.query.parse()*
     Return: ~
         Query Parsed query
 
+                                                   *Query:captures_iterator()*
+Query:captures_iterator({node}, {source}, {start}, {stop})
```

Comment:
random idea could be to just change "iter" to gen, like `function gen_captures() return function gen() ... end end` since it's not a "real" iterator

(might actually be worse though IDK)